src/frontend/components/ScrollToTopButton.js

```diff
@@ -0,0 +1,54 @@
+import CatImage from '../assets/cat.png';
+import UpIcon from '../assets/up.svg'
+
+export default class ScrollToTopbutton {
+    constructor({ $parent }) {
+        this.$target = document.createElement('div');
+        this.$target.classList.add("scroll-top-button")
+
+        $parent.appendChild(this.$target);
+
+        this.render();
+        this.handleEvent();
+    }
+
+    handleEvent() {
+        const throttle = (callback, delay) => {
+            let timerId = null;
+            return (event) => {
+                if (timerId) return;
+                timerId = setTimeout(() => {
+                    callback(event);
+                    timerId = null;
+                }, delay);
+            };
+        };
+
+        const handleScroll = () => {
+            const rootElement = document.documentElement
+            const scrollTotal = rootElement.scrollHeight - rootElement.clientHeight
+
+            if ((rootElement.scrollTop / scrollTotal) > 0.10) {
+                this.$target.classList.add('show-button')
+            } else {
+                this.$target.classList.remove('show-button')
+            }
+        }
+
+        const onScroll = throttle(handleScroll, 100);
```

Comment:
```suggestion
        const DELAY = 100;
        const onScroll = throttle(handleScroll, DELAY);
```
Magic Number를 사용하기보다 적절하게 변수를 네이밍해서 사용하면 더 의미를 정확하게 알 수 있을 것 같습니다 ~